docs/administration-guide/databases/google-analytics.md

```diff
@@ -0,0 +1,76 @@
+## Working with Google Analytics in Metabase
+
+This page provides information on how to create and manage a connection to a [Google Analytics](https://cloud.google.com/analytics) dataset.
+
+## Prerequisites
+
+You will need to have a [Google Cloud Platform](https://cloud.google.com/) account and create the [project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project) you would like to use in Metabase. Consult the Google Cloud Platform documentation on how to [create and manage a project](https://cloud.google.com/resource-manager/docs/creating-managing-projects) if you do not have one.
+
+## Google Cloud Platform: creating OAuth client ID
+
+Follow Google's Cloud Platform Help on [how to create an OAuth client ID](https://support.google.com/cloud/answer/6158849?hl=en) in the console. You will obtain the **Client ID** and **Client Secret** which are required later to populate Metabase's database creation form.
+
+> In order to avoid a `redirect_uri_mismatch` when requesting **Auth Codes** in later steps, ensure to set `Application Type: Desktop App`.
+
+##### Enabling Google Analytics API
+Go to [APIs & Services](https://console.developers.google.com/apis/). The **Enable APIs and Services** button, opens the **API Library**, where you can search for **Google Analytics API** and enable it.
+If further details are needed, you can follow this Google [API Console Help](https://support.google.com/googleapi/answer/6158841?hl=en) article instead.
+
+
+##### Adding the API scopes
+Refer to Google's Analytics API documentation for the required [scopes](https://developers.google.com/identity/protocols/oauth2/scopes).
+Depending on your Google Analytics API version, you may need to use [Google Analytics API, v3](https://developers.google.com/identity/protocols/oauth2/scopes#analytics) scopes or [Analytics Reporting API, v4](https://developers.google.com/identity/protocols/oauth2/scopes#analyticsreporting) ones.
```

Comment:
Currently Metabase only supports v3. See https://github.com/metabase/metabase/issues/12905

Reply:
Removed reference to Google Analytics versions for API scopes.